File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseTipoMantenimiento.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.TipoMantenimientoDTO;

/**
 * Objeto de respuesta con la informacion del tipo de mantenimiento
 * 
 * @author Flor
 *
 */
public class ServerResponseTipoMantenimiento {

	private List<TipoMantenimientoDTO> listaTipoMantenimiento = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseTipoMantenimiento() {
		super();
	}

	public List<TipoMantenimientoDTO> getListaTipoMantenimiento() {
		return listaTipoMantenimiento;
	}

	public void setListaTipoMantenimiento(List<TipoMantenimientoDTO> listaTipoMantenimiento) {
		this.listaTipoMantenimiento = listaTipoMantenimiento;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseTipoMantenimiento.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseTipoMantenimiento
    {
        private List<TipoMantenimientoDTO> _listaTipoMantenimiento;

        public List<TipoMantenimientoDTO> listaTipoMantenimiento
        {
            get { return _listaTipoMantenimiento; }
            set { _listaTipoMantenimiento = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
